modules/nf-core/agat/statistics/main.nf

```diff
@@ -0,0 +1,41 @@
+process AGAT_STATISTICS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::agat=1.0.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/agat:1.0.0--pl5321hdfd78af_0' :
+        'quay.io/biocontainers/agat:1.0.0--pl5321hdfd78af_0  ' }"
+
+    input:
+    tuple val(meta), path(gff)
+
+    output:
+    tuple val(meta), path("*.txt"), emit: stats_file
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def program = args.contains("--basic") ? "agat_sq_stat_basic.pl" : "agat_sp_statistics.pl"
```

Comment:
I would say that the guidelines here will be to have a different module for each of the commands, something such as: `agat/spstatistics` and `agat/sqstatbasic`

Reply:
Also `args` is used to pass the tool arguments itself, in this case `-i` or `--gff`

Reply:
Changes applied. Split in two modules. Included missing ```pytest_modules.yml``` additions.